mysql/mysql.go

```diff
@@ -168,6 +170,23 @@ func Initer() session.Initer {
 			cfg.db = db
 		}
 
+		if cfg.InitTable {
+			q := fmt.Sprintf(`
+				CREATE TABLE sessions (
+					%[1]s      VARCHAR(255) NOT NULL,
+					data       BLOB NOT NULL,
+					expired_at DATETIME NOT NULL,
+					PRIMARY KEY (%[1]s)
+				) DEFAULT CHARSET=utf8`,
```

Comment:
```suggestion
CREATE TABLE sessions (
	%[1]s      VARCHAR(255) NOT NULL,
	data       BLOB NOT NULL,
	expired_at DATETIME NOT NULL,
	PRIMARY KEY (%[1]s)
) DEFAULT CHARSET=utf8`,
```

Besides, you would actually want [`CREATE TABLE IF NOT EXISTS`](https://linuxhint.com/create-table-if-not-exist-mysql/) here, otherwise you'll get "table already exists" error on the second run.

Same for the Postgres.

Reply:
@noneback The indentation suggestion here is on purpose, please fix.